admin_ui/src/components/RowFormErrors.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <div class="errors">
+        <font-awesome-icon icon="exclamation-triangle" />
+        <span>
+            <b> {{ errors }}</b>
+        </span>
+    </div>
```

Comment:
We might have to change this so it's a list.

Something like:

```vue
<p><font-awesome-icon icon="exclamation-triangle" />Error</p>
<ul>
    <li v-for="error in errors" :key="error">{{ error }}</li>
</ul>
```

Reply:
It's a great idea, but as I wrote in previous comments, we only get one error from the server if I understand correctly.